specification/cognitiveservices/data-plane/MetricsAdvisor/preview/v1.0/MetricsAdvisor.json

```diff
@@ -5550,10 +5493,10 @@
       ],
       "properties": {
         "dataSourceParameter": {
-          "$ref": "#/definitions/HttpRequestParameter"
+          "$ref": "#/definitions/AzureTableParameter"
```

Comment:
This does take us back to the discussion about required parameters in a PATCH payload.

Reply:
That's true. While there are a log of xxxPatch have been implemented. So I wonder if it should be turned over at this moment.